test/shell/test_scala_jacocorunner.sh

```diff
@@ -0,0 +1,16 @@
+# shellcheck source=./test_runner.sh
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. "${dir}"/test_runner.sh
+. "${dir}"/test_helper.sh
+runner=$(get_test_runner "${1:-local}")
+
+test_scala_jacocorunner_from_scala_toolchain_passes() {
+  bazel test --extra_toolchains="//manual_test/scala_test_jacocorunner:passing_scala_toolchain" //manual_test/scala_test_jacocorunner:empty_test
+}
+
+test_scala_jacocorunner_from_scala_toolchain_fails() {
+  action_should_fail test --extra_toolchains="//test_expect_failure/scala_test_jacocorunner:failing_scala_toolchain" //test_expect_failure/scala_test_jacocorunner:empty_test
+}
+
+$runner test_scala_jacocorunner_from_scala_toolchain_passes
+$runner test_scala_jacocorunner_from_scala_toolchain_fails
```

Comment:
Small question- does this test fail only when you use the target in the toolchain? If I were to add jacocorunner to the toolchain and not use it in the actual code or maybe even better use it without actually passing it to the actions (still using the _jacocorunner) does it fail? If bazel is completely lazy it won't fail and then this test is good. Otherwise it fail fast and the test is problematic.

Reply:
Good point. I have checked this test with 1. only adding jacocorunner option on scala_toolchain, 2. not using it anywhere.

Unfortunately Bazel fails fast as the target I've given it does not exist.

I'll change the failing tests configuration to use a valid target, so if jacocorunner is not used from the toolchain then it passes.

Reply:
Indeed this is critical, as if jacocorunner from the toolchain is not used, then the positive test passes anyhow (does not detect the problem), but the negative test should also "pass" to detect that jacocorunner from the toolchain was not used.

Reply:
Fixed.

Reply:
Thank you once more... Your comment uncovered another issue. With the negative case fixed it came out that the positive case is also wrong. It's not testing coverage at all, as it runs `bazel test` instead of `bazel coverage`... My bad.

Reply:
Fixed again :)

Reply:
Happy to help. Testing is hard, bash testing is even harder.